pkg/acquisition/modules/s3/s3.go

```diff
@@ -283,23 +308,29 @@ func (s *S3Source) sqsPoll() error {
 			logger.Debugf("Received %d messages from SQS", len(out.Messages))
 			for _, message := range out.Messages {
 				sqsMessagesReceived.WithLabelValues(s.Config.SQSName).Inc()
-				eventBody := S3Event{}
-				err := json.Unmarshal([]byte(*message.Body), &eventBody)
+				bucket, key, err := s.extractBucketAndPrefix(message.Body)
 				if err != nil {
 					logger.Errorf("Error while parsing SQS message: %s", err)
+					//Always delete the message to avoid infinite loop
+					_, err = s.sqsClient.DeleteMessage(&sqs.DeleteMessageInput{
```

Comment:
In the documentation, we may need to specify to use a specific queue only for crowdsec. Because sometimes, users will use one queue for multiple needs, and as crowdsec deletes the messages, it can be a mess.